### PR TITLE
Task/reporting service

### PR DESCRIPTION
**Pull request by esotrope**
This is just step 1 of the refactor
This copies `webapp/*` (minus `webapp/src/main/webapp`) to a new `reporting-service` module
Next step is to add the zuul gateway

Bad news is it is not possibly to see the history of a file after rename on github.
You have to run `git log --follow [filename]` which is not supported through the github ui
(someone made a chrome extension for it though: https://stackoverflow.com/questions/5646174/how-to-make-github-follow-directory-history-after-renames)

git i guess tracks file content changes (not the files) so the history can actually be seen on the file using this command on either file copy.

### CODE REVIEW

**Comment by wtritch** [comment]
I'm not going to review a 269-file copy, but I'll trust you.

File: reporting-service/build.gradle [hunk 1]
```diff
@@ -0,0 +1,39 @@
+apply plugin: 'org.springframework.boot'
+
+createDockerfile {
+    ext.java_opts = ["-Xmx768m"]
```
Comment (wtritch) [inline reporting-service/build.gradle]: Note to future us, once we have the API and the front-end broken out one or both of these can probably be reduced.

**Comment by malaffoon** [comment]
I'm with Will on this ... reviewing a huge set of copied files doesn't work great so i'll trust you. 

One question: i thought we already had a `report-processor` module. Is this new module going to be the mostly-the-webapp thing?

**Comment by esotrope** [comment]
@malaffoon doh - correcting description

**Pull request merged**